tedgoddard/materialAI
# Set OpenAI key

closes #11 

Add setting openai key from UI

#### How to test
- If you have a key already set, when loading the page you should see an obfuscated version of your key in the bottom right of the page
- Remove the key from local storage and reload the page
- Try setting the key using both pressing enter in the text field and by clicking the Set button


When the key is set you'll see...

<img width="287" alt="Screenshot 2023-09-18 at 7 02 26 PM" src="https://github.com/tedgoddard/aiSandbox/assets/1420670/d5a926cd-e881-4390-a641-15437a8a4706">

When the key has not been set you'll see...

<img width="307" alt="Screenshot 2023-09-18 at 7 03 47 PM" src="https://github.com/tedgoddard/aiSandbox/assets/1420670/dc7e8989-1d25-4ec5-8a40-fc9af444f970">
